components/Card/Card.js

```diff
@@ -11,23 +12,30 @@ import {
 } from "./StyledCard.js";
 import DetailsDisplay from "../DetailsDisplay/DetailsDisplay.js";
 
-export default function Card({ routes, refetch }) {
-  async function handleFavoriteClick(routeData) {
-    await putData({ ...routeData, favorite: !routeData.favorite });
-    refetch();
+export default function Card({ routes }) {
+  const { mutate: refetchRoutes } = useSWR("/api/routes");
```

Comment:
I´m not sure if this is a good idea. I have the impression, a second call to `/api/routes` is made here.
But there is another call made before on the page level, right? Please check on this again and possibly pass the refetch function again, as you did before.

Reply:
Hey Thomas,

you was right. Every time when I reloaded my routes or favorite page, I got an error in the console. Now I give the refetchRoutes function down as prop again and the issue is fixed. Thank's for that😃